.gitignore

```diff
@@ -6,3 +6,7 @@ node_modules/
 tsconfig.tsbuildinfo
 package-lock.json
 storybook-static
+.loki/current
+.loki/difference
+.loki/.gitignore
```

Comment:
This is weird, why would you want to ignore a `.gitignore` file? 🤔

Reply:
When you run loki it automatically creates gitignore file, what is not necessary